tutorials/plot_visualize_epochs.py

```diff
@@ -70,7 +71,18 @@
 # interactive mode you can scale and change the colormap with mouse scroll and
 # up/down arrow keys. You can also drag the colorbar with left/right mouse
 # button. Hitting space bar resets the scale.
-epochs.plot_image(278, cmap='interactive')
+epochs.plot_image(278, cmap='interactive', sigma=1., vmin=-250, vmax=250)
+
+###############################################################################
+# We can also give an overview of all channels by calculating  the global
+# field power (or other other aggregation methods). However, combining
+# multiple channel types (e.g., MEG and EEG) in this way is not sensible.
+# Instead, we can use the ``group_by`` parameter. Setting ``group_by`` to
+# 'type' combines channels by type.
+# ``group_by`` can also be used to group channels into arbitrary groups, e.g.
+# regions of interests, by providing a dictionary containing
+# group name -> channel indices mappings.
+epochs.plot_image(combine='gfp', group_by='type', sigma=2., cmap="inferno")
 
 ###############################################################################
 # You also have functions for plotting channelwise information arranged into a
```

Comment:
as the end use:

epochs.plot_topo_image(vmin=-250, vmax=250, title='ERF images', sigma=2.)

it makes the plot much nicer.

Reply:
Turns out there isn't a "title" option yet ... :)

Reply:
For epochs image I mean.